### PR TITLE
Automatically include code from plots in the documentation

### LINKED ISSUE #179: Ease including plots and code for plots in dcumentation
## General

* pyfar version: 0.1.0

## Description

To include a plot and the code for the plot in the Sphinx documentation, we currently write double code, e.g., 

```
>>> pyfar.plot.time(pyfar.signals.noise(256))

.. plot::
    pyfar.plot.time(pyfar.signals.noise(256))
```

we dont need the first line in the example above if we include

`plot_include_source = True`

in *docs/conf.py*

### PR DESCRIPTION
**Pull request by f-brinkmann**
### Which issue(s) are closed by this pull request?

Closes #179

### Changes proposed in this pull request:

- add option to include code from plots in the docstrings and removed redundant code (see #179)
- unified import of pyfar in the docstring examples to `import pyfar as pf`

### CODE REVIEW

**Comment by mberz** [comment]
This seems to remove lack the `>>> ` etc. from the examples, see [here](https://pyfar.readthedocs.io/en/documentation-auto_include_code/pyfar.dsp.html#pyfar.dsp.minimum_phase) for read the docs build

**Comment by f-brinkmann** [comment]
> 
> 
> This seems to remove lack the `>>> ` etc. from the examples, see [here](https://pyfar.readthedocs.io/en/documentation-auto_include_code/pyfar.dsp.html#pyfar.dsp.minimum_phase) for read the docs build

Yes it does, but I didn't bother. The good thing is, copy-pasting works better than in the `>>>` blocks.

**Comment by mberz** [comment]
I actually just realized that the html version does not highlight the >>> anyways, which is pretty nice, so this makes looking for the hide prompt obsolete. 
Anyhow, I kinda like the visual separation aspect of the >>> which allows to set the code apart from return values (we haven't used this so far, but it's nice to have, numpy for example makes use of that.) 
Anyhow, is there a way to re-introduce them. Also for consistency with the other examples?

File: pyfar/classes/audio.py [hunk 1]
```diff
@@ -815,9 +815,9 @@ def __iter__(self):
         --------
         Iterate channels of a :py:func:`Signal`
 
-        >>> import pyfar
+        >>> import pyfar as pf
         >>>
```
Comment (mberz) [inline pyfar/classes/audio.py:819]:
```suggestion
```

**Review by mberz — APPROVED** [review]
Only formatting on one line

**Pull request merged**